.github/workflows/pr-builder.yml

```diff
@@ -27,7 +27,7 @@ jobs:
     strategy:
       matrix:
         node-version: [ lts/* ]
-        pnpm-version: [ 8.7.4 ]
+        pnpm-version: [ 9.5.0 ]
```

Comment:
Can't we use the `lts` version here?

Reply:
There have been some instances in the past where minor/patch pnpm version upgrades containing breaking changes such as lockfile format changes.

Reply:
Fixed with [de6e955](https://github.com/wso2/identity-apps/pull/7238/commits/de6e95516ec9f82b53c795da02be9fcf20a9c738)